### PR TITLE
Removed and references of so-forms from the form and re-added hof. Al…

### PR DESCRIPTION
**Pull request by bendiggle**
…so took the ajax, ajax-edit and reset controllers and put them in into ROTM in the common directory

### CODE REVIEW

**Comment by JoeChapman** [comment]
@bendiggle are there literally not tests?

**Comment by JoeChapman** [comment]
@bendiggle sorry to be a pain, but there are definitely places in this commit that you could chunk into small logical commits. The idea is to make commits as small as possible and retain logically divisions. E.g. You make moving the logger from SO Forms to RTM and updating references to it a single commit.

**Comment by bendiggle** [comment]
@JoeChapman There are only 2 tests, one for the confirm controller and one for the send controller. But both of them don't reference so-forms of hof.

As for the commits, I will do that now. Do you see any more that could be in a separate commit other than the logger?

**Comment by JoeChapman** [comment]
@bendiggle needs rebasing

**Comment by bendiggle** [comment]
@JoeChapman I have rebased and broken down the commit into several. Do they look ok??

**Comment by JoeChapman** [comment]
Much better, cheers Ben 👍 Just waiting for the build to finish

File: assets/scss/app.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 $path: "../img/";
 
-@import "../../node_modules/so-forms/node_modules/hof/node_modules/hmpo-frontend-toolkit/assets/stylesheets/app.scss";
+@import "../../node_modules/hof/node_modules/hmpo-frontend-toolkit/assets/stylesheets/app.scss";
```
Comment (JoeChapman) [inline assets/scss/app.scss]: check this line against GRO

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,9 @@
 language: node_js
 node_js:
-  - "0.12"
+  - "4.4.2"
+
+env:
+  - SERVE_STATIC=true
```
Comment (JoeChapman) [inline .travis.yml]: Don't need this in CI

**Comment by JoeChapman** [comment]
Ok, so this PR has too many commits. It could probably have been done in several PR steps.but I appreciate it's been a pain.

**Pull request merged**